packages/grid/_modules_/grid/components/columnHeaders/GridColumnHeaderItem.tsx

```diff
@@ -123,8 +124,19 @@ export const GridColumnHeaderItem = React.memo(
       [publish],
     );
 
+    const classNames = [classes?.columnHeader];
+
+    if (column.headerClassName) {
+      classNames.push(
+        clsx(
```

Comment:
You can drop the `clsx` if we're going to remove support for arrays.

Reply:
Removed